sw/airborne/modules/computer_vision/lib/vision/image.c

```diff
@@ -578,27 +608,24 @@ void image_show_points(struct image_t *img, struct point_t *points, uint16_t poi
  */
 void image_show_flow(struct image_t *img, struct flow_t *vectors, uint16_t points_cnt, uint8_t subpixel_factor)
 {
+  uint8_t color[4] = {255,255,255,255};
+  int size_crosshair = 5;
+
   // Go through all the points
   for (uint16_t i = 0; i < points_cnt; i++) {
     // Draw a line from the original position with the flow vector
     struct point_t from = {
       vectors[i].pos.x / subpixel_factor,
-      vectors[i].pos.y / subpixel_factor,
-      0,
```

Comment:
what about the other parameters of the `point_t` structure ? shouldn't they be initialized properly ?
if only x,y are needed in most case, maybe an init macro can be added to images.h to hide the other parameters initialization.

Reply:
Good remark! point_t got additional properties in a previous pull request - I believe by Manos. I wonder if this is actually the best way to go, as a point as a mathematical concept would not require properties such as the number of times observed. Perhaps a solution would be an additional struct with a point_t as one of its attributes. However, I feel this is beyond the scope of the current pull request.

Reply:
Can you make an issue about this so that it is addressed in the future?

Reply:
I don't think it's worth an issue